lib/octokit/client/repository_invitations.rb

```diff
@@ -0,0 +1,102 @@
+module Octokit
+  class Client
+
+    # Methods for the Repository Invitations API
+    #
+    # @see https://developer.github.com/v3/repos/invitations/
+    module RepositoryInvitations
+
+      # Invite a user to a repository
+      #
+      # Requires authenticated client
+      #
+      # @param repo [Integer, String, Hash, Repository] A GitHub repository
+      # @param user [String] User GitHub username to add
+      # @return [Sawyer::Resource] The repository invitation
+      # @see https://developer.github.com/v3/repos/invitations/#invite-a-user-to-a-repository
+      def invite_user(repo, user, options = {})
```

Comment:
This is a little ambiguous since methods are global across `Client`. How about `invite_user_to_repository`?